ansible/roles/oso_console_extensions/tasks/main.yml

```diff
@@ -52,3 +46,15 @@
   register: start_build_out
 
 - debug: var=start_build_out
+
+- name: Create edge route for console-extensions url
+  oc_route:
+    name: "{{ osce_appname }}"
+    namespace: "{{ osce_namespace }}"
+    service_name: "{{ osce_appname }}"
+    tls_termination: edge
+    port: 8080
+    host: oso-console-extensions-openshift-infra.d800.free-int.openshiftapps.com
```

Comment:
could this be dynamic so that it won't break devenvs? or should we table that for now

Reply:
yes, I need to find the param to pass for that one. I'll update that